### PR TITLE
Extend subscriptions

### PR DESCRIPTION
**Pull request by Andrei-Dolgolev**

### CODE REVIEW

**Comment by Andrei-Dolgolev** [comment]
- [ ]  update required python -m moonstream.admin.cli subtypes update -i 'ethereum_blockchain' -c 'input:address' 'tag:nfts' | jq
- [ ]  update required python -m moonstream.admin.cli subtypes update -i 'ethereum_txpool' -c 'input:address' 'tag:nfts' | jq
- [ ]  update required python -m moonstream.admin.cli subtypes update -i 'ethereum_whalewatch' -c  | jq

File: backend/moonstream/admin/subscription_types.py [hunk 1]
```diff
@@ -73,6 +76,7 @@ def create_subscription_type(
     id: str,
     name: str,
     description: str,
+    choices: Optional[List[str]],
```
Comment (kompotkot) [inline backend/moonstream/admin/subscription_types.py:79]: = None

File: backend/moonstream/admin/subscription_types.py [hunk 2]
```diff
@@ -266,19 +275,15 @@ def update_subscription_type(
...
     try:
-        bc.delete_resource(
+        new_resource = bc.update_resource(
             token=MOONSTREAM_ADMIN_ACCESS_TOKEN,
             resource_id=brood_resource_id,
+            resource_data={"update": updated_resource_data},
             timeout=BUGOUT_REQUEST_TIMEOUT_SECONDS,
         )
+
     except Exception as e:
```
Comment (kompotkot) [inline backend/moonstream/admin/subscription_types.py:287]:
```
    except BugoutResponseException as e:
        raise MoonstreamHTTPException(status_code=e.status_code, detail=e.detail)
    except Exception as e:
        raise MoonstreamHTTPException(status_code=500, internal_error=e)
```

File: backend/moonstream/data.py [hunk 3]
```diff
@@ -12,6 +12,7 @@ class SubscriptionTypeResourceData(BaseModel):
     id: str
     name: str
     description: str
+    choices: Optional[List[str]]
```
Comment (kompotkot) [inline backend/moonstream/data.py:15]: = None

File: backend/moonstream/providers/bugout.py [hunk 4]
```diff
@@ -19,6 +19,7 @@
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.WARN)
+allowed_tags = ["nfts"]
```
Comment (kompotkot) [inline backend/moonstream/providers/bugout.py:22]: This variable not used nowhere in this PR

**Review by kompotkot — CHANGES_REQUESTED** [review]

**Review by kompotkot — APPROVED** [review]
lg

**Pull request merged**